src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -32,18 +32,45 @@
                 <div v-if="importFromCsvIsEnabled" id="choose-import-from">
                     <div class="col-sm-2 d-inline-block"></div>
                     <div class="btn-group btn-group-toggle" data-toggle="buttons">
-                        <label id="choose-from-list-label" class="btn btn-outline-primary btn-toggle shadow-none active">
+                        <label id="choose-from-list-label"
+                               class="btn btn-outline-primary btn-toggle shadow-none"
+                               :class="reportsOrigin === 'list' ? 'active' : ''">
                             <input type="radio" id="choose-from-list"
                                    v-model="reportsOrigin" value="list"
-                                   autocomplete="off" checked> Choose from list
+                                   autocomplete="off"> Choose from list
                         </label>
-                        <label id="import-from-csv-label" class="btn btn-outline-primary btn-toggle shadow-none">
+                        <label id="import-from-csv-label"
+                               class="btn btn-outline-primary btn-toggle shadow-none"
+                               :class="reportsOrigin === 'csv' ? 'active' : ''">
                             <input type="radio" id="import-from-csv"
                                    v-model="reportsOrigin" value="csv"
                                    autocomplete="off"> Import from csv
                         </label>
                     </div>
                 </div>
+                <div v-if="showImportFromCsv" id="show-import-csv" class="pt-4">
+                    <div class="col-sm-2 d-inline-block"></div>
+                    <div class="custom-file col-sm-6">
+                        <input type="file" class="custom-file-input"
+                               @change="handleImportedFile($event)"
+                               @click="handleClickImport($event)"
+                               accept="text/csv"
+                               id="import-csv"
+                               lang="en">
+                        <label class="custom-file-label" for="import-csv">{{ importedFilename }}</label>
+                    </div>
+                    <div v-if="validationErrors.length"
+                         id="import-validation-errors"
+                         class="text-danger small error-message mt-2 ">
+                        <div class="col-sm-2 d-inline-block"></div>
+                        <div class="col-sm-6 d-inline-block pl-0">
+                            Unable to import from csv:
+                            <div v-for="error in validationErrors" class="import-validation-error ml-2">
```

Comment:
How about if we use standard bootstrap error alert? Since we're using a warning alert when it automatically removes report on change git branch that would be consistent

Reply:
We can use that but with error icon.

Reply:
I think the icon might be overkill - we don't use it for the warnings. And this might be relevant for the discussion on the summary page mockups as to displaying missing deps summary at top of page.